docs/MyFirstPullRequest.md

```diff
@@ -0,0 +1,48 @@
+# My First Pull Request
+
+Okay. You are a member of the Rocket 2 team. You have successfully set up your
+local development environment. You have found an issue that you want to work
+on. Now, what to do?
+
+## Setting up branches
+
+Before you make any changes, you should first set up your own branch. It is
+common convention to name your branch:
+
+```
+<username>/#<issue-number>-<description-of-fix>
+```
+
+So if your issue is [#153 Read from configuration][#153], you would name it
+`rwblickhan/#153-read-from-config`. The name needs to be concise, descriptive,
+and, well, have your name and number, so to speak.
+
+## Before-Pull-Request checklist
+
+- All unit tests pass (`scripts/build_check.sh`)
+- The travis build passes (travis will build your commit when you push it)
+- Your code is presentable and you have **not** committed extra files (think
+  your credentials, IDE config files, cached directories, build directories,
+  etc.)
+- You've written unit tests for the changes you've made, and that they cover
```

Comment:
You should probably note how to check this (i.e. code coverage, what codecov does, etc)